src/blocks/models.py

```diff
@@ -39,8 +40,26 @@ class Meta:
 
 
 class HeadingBlock(blocks.StructBlock):
-    title = blocks.CharBlock(required=True)
-    subtitle = blocks.CharBlock(required=False)
+    title = CharBlock(required=True)
+    title_alignment = ChoiceBlock(
+        default=("center", _("Center")),
+        required=True,
+        choices=[
+            ("start", _("Start")), 
+            ("center", _("Center")), 
+            ("end", _("End"))],
```

Comment:
left, center and right would be better for the users since most users are more familiar with those terms